src/main/rx/expresslrs_impl.h

```diff
@@ -73,6 +73,7 @@ typedef struct elrsReceiver_s {
     bool alreadyFHSS;
     bool alreadyTLMresp;
     bool lockRFmode;
+    bool started;
```

Comment:
Now that there are 4 boolean flags to record state, it feels like bit flags should be used instead.  i.e.  `uint8_t flags`  and 
```
#define ELRS_RX_STATE_FHSS (1 << 0)
#define ELRS_RX_STATE_TLM_RESPONDED (1 << 1)
#define ELRS_RX_STATE_LOCK_RF (1 << 2)
#define ELRS_RX_STATE_STARTED (1 << 3)
```

Thoughts?

Reply:
Good idea, I'll refactor it later